### PR TITLE
feat: Show MLS info on device (WPB-2308)

### PR DESCRIPTION
**Pull request by borichellow**
# What's new in this PR?

Display the MLS info in the device settings
(Screenshots are self-explanatory)

Needs releases with:

- https://github.com/wireapp/kalium/pull/1932

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

| Before | After |
| --------- | --------- | 
| Screenshot 2023-08-02 at 11 59 00 | Screenshot 2023-08-02 at 11 54 11 | 
| Screenshot 2023-08-02 at 11 58 47 | Screenshot 2023-08-02 at 11 53 59 |
|   |   |

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [806](https://10.10.124.134/job/AR-build-pipeline/806/) **failed**.

**Comment by AndroidBob** [comment]
Build [821](https://10.10.124.134/job/AR-build-pipeline/821/) **failed**.

**Comment by AndroidBob** [comment]
Build [823](https://10.10.124.134/job/AR-build-pipeline/823/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.4.0(7199928)(7199928)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.4.0(7199928)(7199928)-staging-compat.apk)

**Review by trOnk12 — APPROVED** [review]

File: app/src/main/kotlin/com/wire/android/ui/authentication/devices/DeviceItem.kt [hunk 1]
```diff
@@ -199,7 +200,24 @@ private fun DeviceItemTexts(
     }
 
     Spacer(modifier = Modifier.height(MaterialTheme.wireDimensions.removeDeviceItemTitleVerticalPadding))
-    val details: String = if (!device.registrationTime.isNullOrBlank()) {
+
+    device.mlsPublicKeys?.values?.firstOrNull()?.let { mlsThumbprint ->
+        Text(
+            style = MaterialTheme.wireTypography.subline01,
+            color = MaterialTheme.wireColorScheme.labelText,
+            maxLines = 1,
```
Comment (vitorhugods) [inline app/src/main/kotlin/com/wire/android/ui/authentication/devices/DeviceItem.kt:208]: Should it really be `maxLines = 1`?

I see it ends up cutting the thumbprint. Shouldn't we make sure that the whole thing is visible?

Comment (borichellow) [reply app/src/main/kotlin/com/wire/android/ui/authentication/devices/DeviceItem.kt:208]: this is `DeviceItem` from the list of all my devices (the first pair of screenshots), if you want to see the whole key - go to device screen :)

**Review by mchenani — APPROVED** [review]
✌️

**Comment by AndroidBob** [comment]
Build [877](https://10.10.124.134/job/AR-build-pipeline/877/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.4.0(7209265)(7209265)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.4.0(7209265)(7209265)-staging-compat.apk)

**Pull request merged**